### ISSUE TITLE
Develop Version Possible Bugs

### ISSUE THREAD

**Issue by scottnix** [ThematicTheme/Thematic#108]
The `<h3 class="menu-toggle">Menu</h3>` gets added to the main menu. Does not matter if Legacy Markup Checkbox is selected, This will break existing themes, example http://scottnix.com/themes/responsivebase/ Of course you can set it to display none, but it would be something to note.

I need to test it more, but it appears the Legacy Toggle Checkbox in Theme Options does not reset, once you click it off, it sticks no matter what and you are stuck in the new html5 mode. At least it was on my multisite, will check regular installs later.

**Comment by falcoraptor** [comment]
re: menu toggle

yes that is true.I kept it in because I wanted to enable the use of the responsive stylesheet also n html4 mode. Hm, I need to revisit that. At the very least make it super clear that this is an addition and existing child themes will need to add styles for it.

Re legacy mode checkbox:

ouch! will need to test that one further. If you can reproduce it, do tell me. I won't have time to test it for a while, I'll put it on the todo list.

**Comment by scottnix** [comment]
On the issue of the title showing, it totally isn't a big deal at all, I will just remove it through CSS, but it is really the only thing I have seen (so far) that will break (for a lack of better words) an existing theme.

As far as the Legacy Mode not being able to be reverted, it is a pretty big issue. I have tested a localhost install, a multisite install and a individual live install, they are all stuck in HTML5 Thematic 2.0 mode now. I tested all the other options just to be a little more useful and they are all working as expected.

**Comment by falcoraptor** [comment]
That is odd, I cannot replicate your problem. I can switch back and forth between legacy mode and html5 mode just fine, both in parent Thematic theme and in a child theme, and in multisite.
- does this happen on an upgraded site or a clean install?
- does it happen with a specific child theme or with just the Parent theme as well?

**Comment by falcoraptor** [comment]
re the menu toggle:
I did add styles for hiding it to the legacy default.css in commit 74f5800
This means that child themes that uses the parent stylesheets won't be affected, but any child theme that do all styling themselves will need to add accommodating styles for hiding the menu toggle h3.

This will need to be clearly communicated in any release docs.

**Comment by falcoraptor** [comment]
re stuck checkbox:
I also tried with recent download of both your responsive base and responsive base plus themes and I can change back and forth no problem. It also works on thematicdev.invistruct.com. 

More details about your setup would be appreciated. To fix it, I need to know in what circumstances it appears.

FWIW I read your notes about layout in the readme of responsive base plus. Yes the 'full-width' body class mimicks the behaviour of the full width page template - the sidebar is gone by default. This also reflects one of the more common requests/questions in the old forums. BUT the sidebar can be re-included by filtering the `thematic_sidebar` and will then appear below the content (not on the page template though, that remains unchanged). The theme is in control there. In fact, in 3.9 the ability was added to [let child themes remove parent page templates](https://core.trac.wordpress.org/ticket/27334) so the child theme really is in complete control.

I also put some articles up on the wiki of the Thematic repo with some explanations regarding [controlling the theme layout](https://github.com/ThematicTheme/Thematic/wiki/Controlling-theme-layout) and controlling the [available layouts](https://github.com/ThematicTheme/Thematic/wiki/Available-layouts). The child theme can easily control which layouts it supports and remove existing or add new to the Theme Customizer.

**Comment by scottnix** [comment]
Now I can't reproduce it, Sorry. I even tried clearing cache before I posted, but I wouldn't think that would even do anything. :(

Everything seems to be working fine now, thank you a ton for the articles, they answer pretty much every question I had. I just wrote those notes because I knew people making themes would need that kind of information, it would be a pain for some projects to try to guess ways the theme could be used just cause the options where there. I will update that info.
